src/components/SearchAndSort/SearchAndSort.js

```diff
@@ -355,7 +355,6 @@ export class SearchAndSort extends Component {
 
   deleteRecord = async record => {
     const { onDelete } = this.props;
-
```

Comment:
Please leave that empty line. Usually, we keep the empty line after destructuring from `props`, `state`, `context` in the exact order.